src/translations/en.json

```
@@ -3089,8 +3092,10 @@
             "dialog": {
               "header": "Configure battery system",
               "entity_para": "Pick sensors which measure energy going into and coming out of the battery in either of {unit}.",
```

Does this remain as the headline above the now 4 sensors for both energy and power?

In that case it should not only enumerate the  units for energy but rather say something like:
```suggestion
              "entity_para": "Pick sensors which measure the electricity flowing in and out of the battery either as energy in {energy_unit} or power in {power_unit}.",
```

---

No I moved this to helpers for each input. Forgot to attach a screenshot

---

OK, then we should still change "energy" to "electricity" in that string and shorten it a bit:

```suggestion
              "entity_para": "Pick sensors that measure the electricity flowing in and out of the battery in either of {unit}.",
```